pset-iterators.js

```diff
@@ -65,7 +69,13 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentsNamedJames = presidents.filter((firstNameJames) => {
+  return firstNameJames.president.includes('James')
+}).map((firstNameJames) => {
+  return firstNameJames.president
+})
 
+console.log(presidentsNamedJames);
```

Comment:
@OcielC  Good Job here chaining together your filter and map method to return the list of presidents whose name begin with James.
One Suggestion
-For this example, you could remove the map method because for this example you were to return an Array of objects and the filter method without the map will give you that outcome. 

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)